vuepress/docs/next/docs/getting-started/multitenancy.md

```diff
@@ -0,0 +1,41 @@
+---
+sidebarDepth: 2
+---
+
+# Entando Multitenancy
+
+## Overview
+
+Entando 7.2 introduces multitenancy to the Entando Platform. An Entando Application can be customized to enable a multitenant architecture where tenants share an infrastructure but are securely informationally isolated. This document provides an overview of multitenancy and its Entando implementation.
+
+## Core Concepts
+
+Software multitenacy describes an architecture in which a single software instance runs on a server and serves multiple tenants. Each tenant constitutes a group of users who share specific access privileges to the software instance. A multitenant software application is designed to provide a common architecture while ensuring the segregation of information.
+
+Entando Multitenancy imposes a shared architecture (e.g. the versions of Entando, K8s and Keycloak) while distributing resources (e.g. CPU, memory) across the primary and secondary tenants. The primary tenant is the user group with full access to the default features and functionalities of an Entando instance. Secondary tenants are user groups who share the privileges and capabilities of the primary tenant with the exception of an active Component Manager. Tenants are informationally isolated from one another with unique data, configuration settings, and user management.
+
+## Architecture
+
+All tenants rely on a single Entando Application instance for core functionality such as the App Builder, App Engine, Keycloak, and Redis. Redis is necessary for cache managment, but not installed by default. See **TODO - add tutorial link** to add Redis to your Entando Application.
+
+To isolate its data and configurations, each tenant is allocated its own subdomain, database schema, Entando Content Deliver Server (CDS) instance, Solr core, and Keycloak client realm. Entando CDS is required to manage static resources external to the Entando App Engine, Solr establishes an external search engine, and the Keycloak realm grants user access.
```

Comment:
subdomain -> domain name. Technically (and we may need to confirm this is done for 7.2), it's the domain name that identifies a tenant but it doesn't have to be a subdomain of the primary tenant - even though that's a common configuration, e.g. for SME.  

Maybe "each tenant is identified by its own domain name and allocated its own database schema, ..."